cetlvast/suites/unittest/test_unbounded_variant.cpp

```diff
@@ -79,53 +96,53 @@ struct side_effect_stats
     }
 };
 
-struct TestBase : rtti_helper<type_id_type<0x0>>
+struct MyBase : rtti_helper<type_id_type<0x0>>
```

Comment:
Renamed these helper classes to not be confused with actual `TestXxx` gtest fixture class (like `TestPmrUnboundedVariant` is).